src/Commands/generate/Generators/Drush/default-methods.twig

```diff
@@ -31,6 +31,7 @@
    * @command {{  machine_name }}:token
    * @aliases token
    *
+   * @filter-output
```

Comment:
should declare a default field instead

Reply:
How do we pick which field is the default field? Assume `id` and let the user change it?